src/lib/components/Collapsible/tests/Collapsible.test.ts

```diff
@@ -1,16 +1,31 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import { render, type RenderResult } from '@testing-library/svelte';
-import { axe } from 'jest-axe';
-import Collapsible from './cmp.svelte';
+import { expect, test } from '@playwright/experimental-ct-svelte';
 
-describe('Collapsible', () => {
-	let rendered: RenderResult<any>;
+import CollapsibleTest from './CollapsibleTest.svelte';
+import { axeViolations } from '../../../helpers/axeTester.js';
```

Comment:
You can add helpers to svelte.config alias: https://kit.svelte.dev/docs/configuration#alias

Reply:
I tried using `$lib` path to start with, but it didn't resolve when running tests. Something about how playwright compiles, it doesn't pass through. Would be good to solve I agree

Reply:
Fixed it, required a change to tsconfig.